assignments/prototypes.js

```diff
@@ -15,13 +15,29 @@
   * dimensions (These represent the character's size in the video game)
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
+const GameObject (){
```

Comment:
When you create constructor functions, you need to declare them either with the function keyword or with a function expression. You could change this to be `function GameObject() {}` or `const GameObject = function() {}`

Reply:
One additional thing to note - the constructor function should take in some parameters to create the new object